parquet/src/file/reader.rs

```diff
@@ -45,23 +45,39 @@ pub trait Length {
     fn len(&self) -> u64;
 }
 
-/// The ChunkReader trait generates readers of chunks of a source.
+/// Generates [`Read`]ers to read chunks of a Parquet data source.
 ///
-/// For more information see [`File::try_clone`]
+/// The Parquet reader uses [`ChunkReader`] to access Parquet data, potentially
+/// decoding in parallel from from the same data source. The returned readers
+/// follow the model of [`File::try_clone`] where mutations of one reader affect
+/// all readers.
```

Comment:
```suggestion
/// The Parquet reader uses [`ChunkReader`] to access Parquet data, allowing
/// multiple decoders to read concurrently from the same file. The returned readers
/// follow the model of [`File::try_clone`] where mutations of one reader affect
/// all readers.
```

If one wants to be pedantic, this can't be used in parallel as it has no synchronisation. Ultimately this should really be an `Rc<RefCell>` but unfortunately this would make the reader not `Send` which unfortunately is a requirement for tokio (even though this is really a limitation of the type system)